packages/react-components/src/organisms/__tests__/UserProfileRole.test.tsx

```diff
@@ -0,0 +1,116 @@
+import { render } from '@testing-library/react';
+import { UserProfileContext } from '@asap-hub/react-context';
+
+import UserProfileRole from '../UserProfileRole';
+
+const defaultProps = {
+  firstName: 'Phillip',
+  labs: [],
+  teams: [],
+  researchInterests: 'These are my research interest',
+  responsibilities: 'these are my responsibilities',
+};
+
+it('generates a heading', () => {
+  const { getByText } = render(<UserProfileRole {...defaultProps} />);
+  expect(getByText(/role.on.asap/i).tagName).toBe('H2');
+});
+
+it('renders a link to team page', () => {
+  const { getAllByRole } = render(
+    <UserProfileRole
+      {...defaultProps}
+      teams={[
+        {
+          id: '42',
+          displayName: 'Team',
+          role: 'Lead PI (Core Leadership)',
+        },
+      ]}
+    />,
+  );
+
+  const links = (getAllByRole('link') as HTMLAnchorElement[]).map(
```

Comment:
If there is only one link couldn't you do this with getByRole and toHaveAttribue('href',